doc/swagger-api-description/api.yaml

```diff
@@ -173,69 +175,159 @@ paths:
           schema:
             $ref: '#/definitions/ErrorModel'
 
-definitions:
+  /clouds:
+    post:
+      # THE PROBLEM: we need to send an existing User and a new Cloud for him. How should we link togeather User and Cloud in the parameters???
```

Comment:
Does this mean that we need to pass User object inside the body of a request? After we will recieve a User object we can access his clouds, can't we?